tools/chef/roles/db-mysql56-client.json

```diff
@@ -3,6 +3,21 @@
   "chef_type": "role",
   "json_class": "Chef::Role",
   "description": "MySQL 5.6 database client",
+  "default_attributes": {
+    "mysql-libs": {
+      "replace_packages": [
+        "mysql55-libs",
```

Comment:
should this have "mysql-libs" in as well, for servers starting from centos defaults